src/modules/intelligence/application/use_cases/translate_article.py

```
@@ -0,0 +1,245 @@
+            content=translated,
+            success=True,
+        )
+
+    def _call_llm(self, prompt: ArticleTranslationPrompt) -> Optional[AnalysisTranslationContent]:
+        """Translate content using LLM service."""
+        try:
+            translated_text = self._llm_service.translate("", prompt.content)
+            if translated_text is None:
+                return None
+            return self._parse_sections(translated_text)
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)